src/nbl/asset/CMeshManipulator.cpp

```diff
@@ -1535,21 +1535,21 @@ bool CMeshManipulator::calcMaxQuantizationError(const SAttribTypeChoice& _srcTyp
 	return true;
 }
 
-core::smart_refctd_ptr<ICPUBuffer> IMeshManipulator::idxBufferFromTriangleStripsToTriangles(const void* _input, size_t _idxCount, E_INDEX_TYPE _idxType)
+core::smart_refctd_ptr<ICPUBuffer> IMeshManipulator::idxBufferFromTriangleStripsToTriangles(const void* _input, size_t _idxCount, E_INDEX_TYPE _idxType, E_INDEX_TYPE _outIndexType)
```

Comment:
what's going on? Why output param is pass-by-value and whatsmore its not even used??

Reply:
my code, we've went over it